src/spring/azext_spring/_params.py

```diff
@@ -247,6 +251,25 @@ def load_arguments(self, _):
                    help='A json file path for the persistent storages to be mounted to the app')
         c.argument('loaded_public_certificate_file', options_list=['--loaded-public-certificate-file', '-f'], type=str,
                    help='A json file path indicates the certificates which would be loaded to app')
+        c.argument('ingress_read_timeout',
+                   type=int,
+                   help='Ingress read timeout value in seconds. Default 300, minimum is 1, maximum is 1800.',
+                   validator=validate_ingress_timeout)
+        c.argument('ingress_send_timeout',
+                   type=int,
+                   help='Ingress send timeout value in seconds. Default 60, minimum is 1, maximum is 1800.',
+                   validator=validate_ingress_send_timeout)
+        c.argument('session_affinity',
+                   arg_type=get_enum_type(SessionAffinity),
+                   help='Ingress session afiinity of app.',
+                   validator=validate_ingress_timeout)
+        c.argument('session_max_age',
+                   type=int,
+                   help='Time until the cookie expires. Minimum is 1 second, maximum is 7 days. If set to 0, the expiration period is equal to the browser session period.',
+                   validator=validate_ingress_session_max_age)
+        c.argument('backend_protocol',
+                   arg_type=get_enum_type(BackendProtocol),
+                   help='Ingress backend protocol of app.')
```

Comment:
Can we use the `for` loop statement for create and update commands to reuse these same parameter definitions?